Decred-Never-Stops.md

```diff
@@ -0,0 +1,54 @@
+| Author | João Paulo Sant'Anna |
+| :---- | :---- |
+| Title | Decred never stops! |
+| Headline  | featured text |
+| Cypherpunktimes | cypherpunktimes.com/...link |
+| Publish Date | 00 Jan 20?? |
+
+| Social Media | Link |
+| :---- | :---- |
+| Twitter | post link… |
+| Twitter | thread link… |
+| Twitter | poll link… |
+| Reddit  | link… |
+| Bison Relay | News feed title |
+| Other platforms | Instagram and TikTok |
+
+Thesis statement 
+
+Decred communitty never stops working, will this be enough to overcome challenges and thrive in the ever-changing crypto market?
+
+Pre-discussion, research and questions:
```

Comment:
Your outline is concise but I wonder if it might be too much for one article. If you find yourself skimming and not adding enough detail you might want to reduce your structure. Personally I'd be included to separate marketing activities from dev activities but it really depends on the level of depth you're aiming to go into.

Reply:
Also, if you have questions you'd like to ask the community, you should add them here and then share them in the writers channel. You might have to bring the responses over in some instances. Adding them as a comment like this one will help to keep them organised and collated.

Reply:
thanks. I'll try to make it more concise. I'm asking for quotes about this topic for some member separately. Arlready sent messages to Tivra, Exitus and Jake.